Applications/PlazaDetails/ServiceMaturity.qmd

```diff
@@ -42,43 +43,43 @@ When a service is labelled as either verified or validated, they mark the same l
 ### Level 3a: Verified
 
 - A comprehensive functional and integration test should be possible.
-  
-- There should be advanced logging that could help while debugging.
-  
-- Service metadata should include information on detailed descriptions of the services, their parameters and a link to a publication that supports the methodology adopted. An example of executing service expected outcome should be provided in a similar manner to that of incubating service.
+  
+- Advanced logging should help while debugging.
+  
+- Service metadata should include information on detailed descriptions of the services, their parameters and a link to a publication that supports the methodology adopted. An example of the expected outcome from running service should be provided similarly to that of incubating service.
 
 - Approximate cost estimation on a larger scale should be presented.
 
-:::{callout-note}
+::: {.callout-note}
 
-Please mention or provide a report to the support team if there exist any constraints/limitations with the services that should be considered.
+Please mention or provide a report to the support team if any constraints/limitations with the services exist that should be considered.
 
 :::
 
 ### Level 3b: Validated
 
-- All the criteria mentioned for the verified level are applicable to this level, along with the additional criteria that the validation report should be provided either as a separate document to the support team or a non-expiring link.
+- All the criteria mentioned for the verified level apply to this level. Additionally, the validation report must be provided either as a separate document to the support team or as a non-expiring link.
 
-:::{callout-note}
+::: {.callout-note}
 
-Although services can be either of verified or validated type, i.e. if a service satisfies all the criteria mentioned under verified but does not provide a validation report despite being relevant to them will be labelled as incubating.
+Although services can be either verified or validated, i.e., if a service satisfies all the criteria mentioned under verified but does not provide a validation report despite being relevant to them, it will be labelled as incubating.
```

Comment:
isn't it then a cleaner solution to have the presence of the validation report as a criteria as well?

Then this explanation is no longer needed?